fjord/analytics/tests/test_views.py

```diff
@@ -58,6 +58,9 @@ def test_front_page(self):
         eq_(pq('.block.count strong').text(), '7')
         eq_(len(pq('li.opinion')), 7)
 
+        assert '&amp;raquo;' not in r.content
+        assert '&raquo;' in r.content
```

Comment:
Plus I'd put this test in `fjord/base/tests/` since it tests the l10n stuff which lives there.